gostream/codec/x264/utils.go

```diff
@@ -27,3 +38,15 @@ func (f *factory) New(width, height, keyFrameInterval int, logger logging.Logger
 func (f *factory) MIMEType() string {
 	return "video/H264"
 }
+
+// calcBitrateFromResolution calculates the bitrate based on the given resolution and framerate.
+func calcBitrateFromResolution(width, height int, framerate float32) int {
+	bitrate := int(float32(width) * float32(height) * framerate * encodeCompressionRatio)
```

Comment:
Does bitrate scale linearly with width and height? Apparently the answer is complicated https://www.reddit.com/r/explainlikeimfive/comments/hlrwcu/eli5_what_is_the_relationship_between_bit_rate/

Reply:
I am currently scaling the bitrate linearly with the number of pixels. 

Interesting graphs in the thread that show logarithmic quality gains for a bitrate for a given size/fps. Will try to verify that we are hitting these sweet-spots. The amount of scene change also comes into play here so it gets quite complicated.

Reply:
If you look at the example bitrates in the description we seem to be pretty bang on to the recommended settings linked in the jira ticket: https://support.google.com/youtube/answer/2853702?hl=en